backend/visualization/views.py

```diff
@@ -57,6 +57,22 @@ def post(self, request, *args, **kwargs):
             raise APIException(detail='could not generate geo data')
 
 
+class MapCentroidView(APIView):
+    '''
+    Retrieve the centroid of a field in a corpus.
+    '''
+
+    permission_classes = [IsAuthenticated, CanSearchCorpus, CanSearchTags]
```

Comment:
I don't think we should require the `IsAuthenticated` permission here. We want this request to be possible also for users who aren't logged in, as long as they have corpus permissions. I'm not quite sure about the `CanSearchTags` permission, my hunch is that this is also unnecessary.

Reply:
I see that the other visualizations have the `CanSearchTags` permission active, so probably fine to keep this here, too. Probably best to copy the permissions from the other visualization views, i.e., `[CanSearchCorpus, CanSearchTags]`

Reply:
Updated